### PR TITLE
Configure TLS clients to use a preferred set of ciphersuites.

### PR DESCRIPTION
**Pull request by mpuncel**

### CODE REVIEW

**Comment by mpuncel** [comment]
cc @csstaub @mcpherrinm

**Comment by rudle** [comment]
I'm no cyber cipher critic, but this change seems OK to me.

File: pkg/util/net/tls.go [hunk 1]
```diff
@@ -39,6 +39,15 @@ func GetTLSConfig(certFile, keyFile, caFile string) (*tls.Config, error) {
 		Certificates: certs,
 		ClientCAs:    cas,
 		RootCAs:      cas,
+		CipherSuites: Safe,
 	}
```
Comment (csstaub) [inline pkg/util/net/tls.go]: You might want to add:
* `PreferServerCipherSuites: true` to make sure the ordering of the server is honored
* `MinVersion:   tls.VersionTLS12` to set the minimum TLS version to 1.2

File: pkg/util/net/tls.go [hunk 2]
```diff
@@ -39,6 +39,15 @@ func GetTLSConfig(certFile, keyFile, caFile string) (*tls.Config, error) {
...
 	return tlsConfig, nil
 }
+
+var Safe = []uint16{
+	tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+	tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
+	tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
+	tls.TLS_RSA_WITH_AES_128_CBC_SHA,
+	tls.TLS_RSA_WITH_AES_256_CBC_SHA,
```
Comment (csstaub) [inline pkg/util/net/tls.go]: Ghostunnel has a set of safe cipher suites if you want to add more:
https://github.com/square/ghostunnel/blob/master/tls.go

**Pull request merged**